services/src/main/java/org/apache/druid/cli/ServerRunnable.java

```diff
@@ -49,6 +53,8 @@
  */
 public abstract class ServerRunnable extends GuiceRunnable
 {
+  public static final String CENTRALIZED_DATASOURCE_SCHEMA_ENABLED = CentralizedDatasourceSchemaConfig.PROPERTY_PREFIX + ".enabled";
```

Comment:
Line exceeds limit of 120 chars:
```suggestion
  public static final String CENTRALIZED_DATASOURCE_SCHEMA_ENABLED
      = CentralizedDatasourceSchemaConfig.PROPERTY_PREFIX + ".enabled";
```